### DISCUSSION TITLE
Adding Organization (O) and Organizational Unit (OU) to the issued to

### DISCUSSION THREAD

**Discussion by willdovetx** [polymind-inc/acmebot#548] [Q&A] [answered]
Is there a way to add the Organization and Organizational Unit to the certificate issued to section?  

![image](https://user-images.githubusercontent.com/56144799/209407440-2b64d6d9-a5d2-4837-84c7-4907c2d75187.png)

Thanks,

**Comment by shibayan** [comment, marked as answer]
Let's Encrypt does not allow organizations to be specified.

https://community.letsencrypt.org/t/organization-o-and-organizational-unit-ou-in-field-issued-to/5236
